src/layouts/summit-layout.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2018 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import React, {useEffect} from 'react'
+import { connect } from 'react-redux';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import {getAllFromSummit} from '../actions/base-actions';
+import PrimaryLayout from "./primary-layout";
+import PlanSelectionPage from "../pages/plan-selection-page";
+
+const SummitLayout = ({summit, loading, match, speaker, location, getAllFromSummit}) => {
+    useEffect(() => {
+        let summitSlug = match.params.summit_slug;
```

Comment:
could we redeclared this outside as useMemo ?
const summitSlug = useMemo(() =>  match.params.summit_slug, [ match?.params?.summit_slug])